tst/regression/utils/test_case.py

```diff
@@ -207,20 +208,21 @@ def Run(self):
         print(run_command)
         sys.stdout.flush()
         try:
-            subprocess.check_call(run_command)
+            proc = subprocess.run(run_command, check=True, stdout=PIPE, stderr=PIPE)
+            return proc.stdout
```

Comment:
Instead of returning, can you simply add a stdouts attribute to the parameters class at the top of this file. You can then simply append to self.parameters.stdouts, and then there is no reason to have to pass it in to analse function.

Reply:
Great point! That make this much cleaner.